dev/codemeta.py

```diff
@@ -0,0 +1,23 @@
+"""Utility script to work with the codemeta.json file."""
+import json
+
+FILENAME = "../codemeta.json"
+
+# add potentially missing content
+with open(FILENAME, "r") as f:
+    data = json.load(f)
+data["maintainer"] = data["author"][0]
+data["readme"] = "https://gammapy.org"
+data["issuetracker"] = "https://github.com/gammapy/gammapy/issues"
```

Comment:
`issueTracker` instead of `issuetracker`

Reply:
ok, done